src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,11 +4,14 @@
  * No ending dot
  */
 public class OtherClass {
```

Comment:
**Reporter**: Findbugs
**Rule**: NM_FIELD_NAMING_CONVENTION
**Severity**: INFO
**File**: se/bjurr/violations/lib/example/OtherClass.java L6
**Source**: se.bjurr.violations.lib.example.OtherClass

Field names should start with a lower case letter&#10;&#10;&#10;  &lt;p&gt;&#10;Names of fields that are not final should be in mixed case with a lowercase first letter and the first letters of subsequent words capitalized.&#10;&lt;/p&gt;&#10;

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2096885470>*